### DISCUSSION TITLE
Usage of unfiltered backprojection

### DISCUSSION THREAD

**Discussion by Olin1461** [astra-toolbox/astra-toolbox#446] [Q&A] [answered]
I have read the example of using FDK for 3D cone beam reconstruction. Is it possible to use unfiltered backprojection for a 3D cone beam case (any way to turn off the filters) in the Astra toolbox?

Is BP3D_CUDA the way to get unfiltered backprojection of the projection data, and does this algorithm support 3D cone beam geometry?

Thank you for your time.

**Comment by askorikov** [comment, marked as answer]
Hi! That's right, BP3D_CUDA is the way to go. It does support cone geometry.
